GoogleDataTransport/GDTCORTests/Common/Categories/GDTCORRegistrar+Testing.m

```diff
@@ -21,10 +21,10 @@
 @implementation GDTCORRegistrar (Testing)
 
 - (void)reset {
+  [self.targetToPrioritizer removeAllObjects];
+  [self.targetToUploader removeAllObjects];
   dispatch_sync(self.registrarQueue, ^{
-    [self.targetToPrioritizer removeAllObjects];
-    [self.targetToUploader removeAllObjects];
-  });
+                });
```

Comment:
Now that this block is empty, can the whole dispatch statement be removed?

Reply:
I think it is needed to make sure all background tasks are finished before the method exits.